docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
@margaretmoser we hadn't considered channel, broker, center so keep them coming! I may have accidentally implied that repeating ones wasn't productive but it actually is. Others may rally around ones we initially dismissed, which we did mostly because we felt the community wouldn't like it.